azure-mgmt-website/src/main/java/com/microsoft/azure/management/website/DeploymentSlot.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.website;
+
+import com.microsoft.azure.management.resources.fluentcore.arm.models.IndependentChildResource;
+import com.microsoft.azure.management.resources.fluentcore.model.Updatable;
+
+/**
+ * An immutable client-side representation of an Azure Web App deployment slot.
+ */
+public interface DeploymentSlot extends
+        IndependentChildResource,
+        WebAppBase<DeploymentSlot>,
+        Updatable<DeploymentSlot.Update> {
+
+    WebApp parent();
+
+    /**************************************************************
+     * Fluent interfaces to provision a deployment slot
+     **************************************************************/
+
+    /**
+     * Container interface for all the definitions that need to be implemented.
+     */
+    interface Definition extends
+            DefinitionStages.Blank,
+            DefinitionStages.WithConfiguration,
+            WebAppBase.Definition<DeploymentSlot> {
+    }
+
+    /**
+     * Grouping of all the deployment slot definition stages.
+     */
+    interface DefinitionStages {
+        /**
+         * The first stage of the deployment slot definition.
+         */
+        interface Blank extends WithConfiguration {
+        }
+
+        /**
+         * A deployment slot definition allowing the configuration to clone from to be specified.
+         */
+        interface WithConfiguration {
+            /**
+             * Create the deployment slot with brand new site configurations.
```

Comment:
descriptions of methods are written in 3rd person declarative mood. i.e. "Creates ...." etc. Theer are multiople instances of this in these files.